pyproject.toml

```diff
@@ -22,6 +22,9 @@ flake8 = "^4.0.1"
 black = "^22.1.0"
 pytest = "^6.2.5"
 pytest-cov = "^3.0.0"
+mypy = "^0.971"
+sklearn = "^0.0"
```

Comment:
sklearn and pandar are needed for the e2e tests to run the example model, thus adding them as development dependencies.

Reply:
gebius